Aspire Budgeting/AppDelegate.swift

```diff
@@ -7,29 +7,40 @@ import UIKit
 
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/02f411f4e99e9c92eda83f52b2dcb91b2ca8561d/Aspire Budgeting/AppDelegate.swift#L9"></span>Consider using final for this class or use a struct (final_class)</td>
    </tr>
  </tbody>
</table>